test/system/__snapshots__/basic.test.ts.snap

```diff
@@ -6,9 +6,11 @@ exports[`\`snyk-api-import help <...>\` Shows help text as expected 1`] = `
 Kick off API powered import
 
 Commands:
-  index.js import     Kick off API powered import         [default] [aliases: i]
-  index.js orgs:data  Generate data required for Orgs to be created via API by
-                      mirroring a given source.
+  index.js import       Kick off API powered import       [default] [aliases: i]
+  index.js orgs:create  Create the Orgs in Snyk based on data file generated
```

Comment:
Update the `docs` as well with how to use this + filename that will be generated + example format